packages/genesys-spark-components/src/components/stable/gux-form-field/gux-form-field.scss

```diff
@@ -5,3 +5,11 @@
   line-height: var(--gse-ui-formControl-label-text-lineHeight);
   color: var(--gse-ui-formControl-label-labelColor);
 }
+
+@mixin gse-ui-formControl-label-bold {
+  font-family: var(--gse-ui-formControl-label-textBold-fontFamily);
+  font-size: var(--gse-ui-formControl-label-textBold-fontSize);
+  font-weight: var(--gse-ui-formControl-label-textBold-fontWeight);
+  line-height: var(--gse-ui-formControl-label-textBold-lineHeight);
+  color: var(--gse-ui-formControl-label-labelColor);
+}
```

Comment:
Should this styling also be applied to the slotted labels of `gux-form-field-checkbox` and `gux-form-field-radio` ?

Reply:
I confirmed with Jackie, the group labels for checkbox and radio groups will be bold but the individual labels will be regular weight.